README.md

```diff
@@ -1 +1,47 @@
-# to-do-list
\ No newline at end of file
+# To Do List
+
+It is a To Do List on which the user can add, delete and edit task to be completed.
+
+### Screenshot
+
+![todo](https://user-images.githubusercontent.com/38283436149143676-2d9db819-800c-418c-9bce-5aa114c7adb3.PNG)
```

Comment:
- [ ] in the repo this screenshot is visible as a link instead please make sure it is an actual picture in the readme file.